consensus/src/simplex/mocks/application.rs

```diff
@@ -1,7 +1,7 @@
 use super::relay::Relay;
-use crate::{simplex::Context, Automaton as Au, Committer as Co, Proof, Relay as Re};
+use crate::{simplex::Context, Automaton as Au, Committer as Co, DigestBytes, Proof, Relay as Re};
```

Comment:
I think we can actually just leave the alias as `Digest` for now (rather than `DigestBytes`)? That should make this change much smaller.

Reply:
I agree !